spec/views/post_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'rails_helper'
+require 'rails-controller-testing'
+
+RSpec.describe Post, type: :system do
+  describe 'index' do
```

Comment:
- [ ] In the user post index page, kindly add these tests either as the requirements ask:
- Tests that asserts you can see a section for pagination if there are more posts than fit on the view.
- Tests that asserts you can see the number of posts the user has written.

Reply:
@tugsem How are you 

Can you tell me how I can create this test 
 Tests that asserts you can see a section for pagination if there are more posts than fit on the view.